src/coreclr/vm/appdomain.hpp

```diff
@@ -1549,8 +1549,6 @@ class AppDomain final
 
 #endif
 
-#if defined(FEATURE_TIERED_COMPILATION)
```

Comment:
Why are we making TieredCompilationManager bits non-conditional? Just for simplicity? I'm not opposed to it at all, just want to understand. On the interpreter side of thing we've been typically making stuff like this feature-gated.

Reply:
Yes, for simplicity. The class is a nop when `FEATURE_TIERED_COMPILATION` isn't set. The feature defines haven't been done very consistently and so code has become very complicated and trying to remove is more impactful than just making it a nop.